sunpy/coordinates/transformations.py

```diff
@@ -197,52 +213,64 @@ def hpc_to_hpc(heliopcoord, heliopframe):
     return hpc
 
 
-# Make a transformation graph for the documentation, borrowed lovingly from
-# Astropy.
+def _make_rotation_matrix(start_representation, end_representation):
+    """
+    Return the matrix for the direct rotation from one representation to a second representation.
+    The representations need not be normalized first.
+    """
+    A = start_representation.to_cartesian()
+    B = end_representation.to_cartesian()
+    rotation_axis = A.cross(B)
+    rotation_angle = -np.arccos(A.dot(B) / (A.norm() * B.norm()))  # negation is required
+
+    # This line works around some input/output quirks of Astropy's rotation_matrix()
+    matrix = np.array(rotation_matrix(rotation_angle, rotation_axis.xyz.value.tolist()))
+    return matrix
+
 
+# The Sun's north pole is oriented RA=286.13 deg, dec=63.87 deg in ICRS, and thus HCRS as well
+# (See Archinal et al. 2011,
+#   "Report of the IAU Working Group on Cartographic Coordinates and Rotational Elements: 2009")
+# The orientation of the north pole in ICRS/HCRS is assumed to be constant in time
+_solar_north_pole_hcrs = UnitSphericalRepresentation(lon=286.13*u.deg, lat=63.87*u.deg)
 
-def _make_transform_graph_docs():
+
+# Calculate the rotation matrix to de-tilt the Sun's rotation axis to be parallel to the Z axis
+_sun_detilt_matrix = _make_rotation_matrix(_solar_north_pole_hcrs,
+                                           CartesianRepresentation(0, 0, 1))
+
+
+@frame_transform_graph.transform(DynamicMatrixTransform, HCRS, HeliographicStonyhurst)
+def hcrs_to_hgs(hcrscoord, hgsframe):
     """
-    Generates a string for use with the coordinate package's docstring
-    to show the available transforms and coordinate systems
+    Convert from HCRS to Heliographic Stonyhurst.
```

Comment:
I wonder if it might be wise to write a paragraph in this docstring describing the transformation in words? The code itself is beautifully terse... but as  @Cadair I'm sure can attest to, it's not at *all* easy to wrap your brain around.  So if anyone ever wants to modify this, having a written description might be helpful.

Alternatively, such a description *might* fit better in the docs (you could include diagrams if desired there), although that might be overkill.